internal/pkg/config/v1/config.go

```diff
@@ -62,7 +62,11 @@ var (
 	)
 	RequireOnPremLoginErr = errors.NewErrorWithSuggestions(
 		"you must log in to Confluent Platform to use this command",
-		"Log in with `confluent login --url <mds-url>`.",
+		"Log in to Confluent Platform with `confluent login --url <mds-url>`.",
+	)
+	RunningOnPremCommandInCloudErr = errors.NewErrorWithSuggestions(
+		"this is not a Confluent Cloud command. You must log in to Confluent Platform to use this command",
+		"Log in to Confluent Platform with `confluent login --url <mds-url>`.\n"+`Run command with "--help" to see available commands.`,
```

Comment:
```suggestion
		"Log in to Confluent Platform with `confluent login --url <mds-url>`.\n"+`Run this command with "--help" to see available commands.`,
```

Or maybe `Use the "--help" flag to...`.

Reply:
i like `use the help flag` better, cz if you're running `confluent cluster --help` it's no point anyways